### DISCUSSION TITLE
Cluster Scaling

### DISCUSSION THREAD

**Discussion by vasilytray** [autobase-tech/autobase#532] [Q&A] [answered]
When scaling a cluster, is it possible to add a new node to the etcd service?

**Comment by vitabaks** [comment, marked as answer]
At the moment, automation does not scale the DCS cluster, but it is [planned](https://github.com/vitabaks/postgresql_cluster/issues/503) for future releases.

scaling is currently implemented for postgres and haproxy balancers

https://github.com/vitabaks/postgresql_cluster#cluster-scaling
